### PR TITLE
reconfiguring vars structure live-conf case

### PR DESCRIPTION
**Pull request by naumvd95**

### CODE REVIEW

File: servers/partner-ci/9.0/manila.yaml [hunk 1]
```diff
@@ -275,8 +257,18 @@
...
+      - shell:
+          if [ ! z ${SPECIFIC_PROPS} ]; then
+            > specific.prop
+            for param in ${SPECIFIC_PROPS}; do echo $param >> specific.prop; done
+      - shell:
+          if [ ! z ${{COMMON_PROPS}} ]; then
+            > common.prop
+            for param in ${COMMON_PROPS}; do echo $param >> common.prop; done
+      - inject:
+          properties-file: specific.prop
```
Comment (rkhozinov) [inline servers/partner-ci/9.0/manila.yaml]: .prop -> .properties

File: servers/partner-ci/9.0/manila.yaml [hunk 2]
```diff
@@ -275,8 +257,18 @@
       - test-property
     builders:
       - get-refs
+      - shell:
+          if [ ! z ${SPECIFIC_PROPS} ]; then
```
Comment (rkhozinov) [inline servers/partner-ci/9.0/manila.yaml]: #!/bin/bash

File: servers/partner-ci/common.yaml [hunk 3]
```diff
@@ -45,6 +45,34 @@
           name: SKIP_VOTE_FAILED
           default: '{skip_vote_failed}'
           description: "configure gerrit voting"
+      - bool:
+          name: KVM_USE
+          default: '{obj:kvm_use}'
+          description: 'turns on kvm_mode for an openstack cluster'
+      - bool:
+          name: FORCE_DISABLE_UPDATES
```
Comment (rkhozinov) [inline servers/partner-ci/common.yaml]: Rarely used variables we can move to the $COMMON_PROPS

File: servers/partner-ci/common.yaml [hunk 4]
```diff
@@ -86,35 +114,30 @@
...
+          name: PLUGIN_VERSION
+          default: '{plugin-version}'
+      - text:
+          name: COMMON_PROPS
+            default: |
+            SNAPSHOT_ARTIFACTS_FILE=snapshots.params
+            OPENSTACK_RELEASE=Ubuntu
+            NODES_COUNT=10
+            DESTROY_ENV_AFTER=0
+            ADMIN_NODE_MEMORY=4096
```
Comment (rkhozinov) [inline servers/partner-ci/common.yaml:111]: need to redefine to null variables for changing amount of a node resources for projects, where DEVOPS_TEMPLATE is used

**Comment by rkhozinov** [comment]
`ENABLE_MOS_`* variables should be also moved to **SPECIFIC_** and **COMMON_ PROPS**

File: servers/partner-ci/9.0/manila.yaml (hunk 1 above)
Comment (naumvd95) [reply servers/partner-ci/9.0/manila.yaml]: done

File: servers/partner-ci/9.0/manila.yaml (hunk 2 above)
Comment (naumvd95) [reply servers/partner-ci/9.0/manila.yaml]: done

File: servers/partner-ci/9.0/manila.yaml [hunk 5]
```diff
@@ -298,20 +309,29 @@
             - inject:
                 properties-content: |
                   ENABLE_MOS_CENTOS_OS=true
-                  ENABLE_MOS_CENTOS_PROPOSED=true
+                  ENABLE_MOS_CENTOS_PROPOSED=false
```
Comment (rkhozinov) [inline servers/partner-ci/9.0/manila.yaml]: ENABLE_MOS_CENTOS_PROPOSED=$ENABLE_PROPOSED

File: servers/partner-ci/9.0/manila.yaml [hunk 6]
```diff
@@ -275,8 +257,29 @@
...
       - inject:
-          properties-file: defaults.prop
+          properties-file: specific.properties
+      - conditional-step:
+          condition-kind: shell
+          condition-command: |
+            [[ "${{SPECIFIC_DEVOPS_TEMPLATE}}" == "true" ]]
+          steps:
+            - inject:
+                properties-content: |
```
Comment (rkhozinov) [inline servers/partner-ci/9.0/manila.yaml]: move to specific and initialize 
```
ADMIN_NODE_MEMORY=4096		
SLAVE_NODE_MEMORY=4096		
ADMIN_NODE_CPU=4		
SLAVE_NODE_CPU=4
```

File: servers/partner-ci/common.yaml (hunk 3 above)
Comment (naumvd95) [reply servers/partner-ci/common.yaml]: done

File: servers/partner-ci/common.yaml (hunk 4 above)
Comment (naumvd95) [reply servers/partner-ci/common.yaml:111]: done

File: servers/partner-ci/9.0/manila.yaml (hunk 5 above)
Comment (naumvd95) [reply servers/partner-ci/9.0/manila.yaml]: done

File: servers/partner-ci/9.0/manila.yaml (hunk 6 above)
Comment (naumvd95) [reply servers/partner-ci/9.0/manila.yaml]: rejected

File: servers/partner-ci/common.yaml [hunk 7]
```diff
@@ -142,6 +118,13 @@
             SNAPSHOT_ARTIFACTS_FILE=snapshots.params
             ESXI_PASSWORD=swordfish
             ESXI_USER=root
+            FORCE_DISABLE_UPDATES=false
+            DRIVER_ENABLE_ACPI=false
+            MAKE_SNAPSHOT=false
+            FUEL_STATS_ENABLED=false
+            ALWAYS_CREATE_DIAGNOSTIC_SNAPSHOT=false
```
Comment (rkhozinov) [inline servers/partner-ci/common.yaml]: true

File: servers/partner-ci/common.yaml [hunk 8]
```diff
@@ -142,6 +118,13 @@
             SNAPSHOT_ARTIFACTS_FILE=snapshots.params
             ESXI_PASSWORD=swordfish
             ESXI_USER=root
+            FORCE_DISABLE_UPDATES=false
+            DRIVER_ENABLE_ACPI=false
+            MAKE_SNAPSHOT=false
```
Comment (rkhozinov) [inline servers/partner-ci/common.yaml]: true

File: servers/partner-ci/common.yaml (hunk 4 above)
Comment (rkhozinov) [reply servers/partner-ci/common.yaml:111]: ok

File: servers/partner-ci/common.yaml [hunk 9]
```diff
@@ -120,9 +120,9 @@
             ESXI_USER=root
             FORCE_DISABLE_UPDATES=false
             DRIVER_ENABLE_ACPI=true
```
Comment (rkhozinov) [inline servers/partner-ci/common.yaml:122]: ` DRIVER_ENABLE_ACPI=false`

**Review by rkhozinov — CHANGES_REQUESTED** [review]

**Review by rkhozinov — APPROVED** [review]

**Pull request merged**